src/group_theory/quotient_group.lean

```diff
@@ -412,4 +413,25 @@ monoid_hom.to_mul_equiv
 
 end third_iso_thm
 
+
+section trivial
+
+/-These first few lemmas should maybe go somewhere else, where?-/
+
+lemma quot_by_top_is_subsingleton : subsingleton (quotient_group.quotient (⊤ : subgroup G)) :=
+trunc.subsingleton
+
+lemma setoid.eq_top_iff {α : Type*} {s : setoid α} : s = (⊤ : setoid α) ↔ ∀ x y : α, s.rel x y :=
+by simp [eq_top_iff, setoid.le_def, setoid.top_def, top_apply]
```

Comment:
This does not belong in the `quotient_group` namespace, and probably belongs in `setoid.lean`

Reply:
Sure I put it in `setoid/basic` right after  `bot_def`.